pandas/core/categorical.py

```diff
@@ -631,7 +633,7 @@ def as_ordered(self, inplace=False):
            Whether or not to set the ordered attribute inplace or return a copy
            of this categorical with ordered set to True
         """
-        return self.set_ordered(True, inplace=inplace)
+        return self.set_ordered(True, inplace=_validate_bool_type(inplace, 'inplace'))
```

Comment:
This line is too long (PEP8, that is the reason travis is failing). You can put the validation of inplace on its own line